openlibrary/templates/account/follows.html

```diff
@@ -1,21 +1,25 @@
-$def with (user, follows=None, manage=False)
+$def with (user, num_found, results_per_page, follows=None, manage=False)
```

Comment:
I opted not to put defaults here so that the onus is strictly on account.py to populate them, and the page size passed to this template will always equal the page size being used in `get_following`. Please let me know if defaults would be preferable!